modules/generative-palm/config/class_settings.go

```diff
@@ -33,14 +34,18 @@ const (
 )
 
 var (
-	DefaultPaLMApiEndpoint        = "us-central1-aiplatform.googleapis.com"
-	DefaultPaLMModel              = "chat-bison"
-	DefaultPaLMTemperature        = 0.2
-	DefaultTokenLimit             = 256
-	DefaultPaLMTopP               = 0.95
-	DefaultPaLMTopK               = 40
-	DefaulGenerativeAIApiEndpoint = "generativelanguage.googleapis.com"
-	DefaulGenerativeAIModelID     = "chat-bison-001"
+	DefaultPaLMApiEndpoint            = "us-central1-aiplatform.googleapis.com"
+	DefaultPaLMModel                  = "chat-bison"
+	DefaultPaLMRegion                 = "us-central1"
+	DefaultPaLMTemperature            = 1.0
```

Comment:
**question:** `DefaultPaLMTemperature` is intended yo be updated ? it was `2` before